tools/pot/docs/BestPractices.md

```diff
@@ -97,10 +97,10 @@ A fragment of Accuracy-aware Quantization configuration with default settings is
 
 Since the Accuracy-aware Quantization calls the Default Quantization at the first step it means that all the parameters of the latter one are also valid and can be applied to the accuracy-aware scenario.
 
-> **NOTE**: In general case, possible speedup after applying the Accuracy-aware Quantization algorithm is less than after the Default Quantization when the model gets fully quantized.
+> **NOTE**: In general, possible speedup after applying the Accuracy-aware Quantization algorithm is less than after the Default Quantization when the model gets fully quantized.
 
 ### Reducing the performance gap of Accuracy-aware Quantization
-To improve model performance after Accuracy-aware Quantization, you can try the `"tune_hyperparams"` setting and set it to `True`. It will enable searching for optimal quantization parameters before reverting layers to the "backup" precision. Note, that this can increase the overall quantization time.
+To improve model performance after Accuracy-aware Quantization try the `"tune_hyperparams"` setting and set it to `True`. It will enable searching for optimal quantization parameters before reverting layers to the "backup" precision. Yet, remember that this can increase the overall quantization time.
```

Comment:
```suggestion
To improve model performance after Accuracy-aware Quantization, try the `"tune_hyperparams"` setting and set it to `True`. It will enable searching for optimal quantization parameters before reverting layers to the "backup" precision. Yet, remember that this can increase the overall quantization time.
```